### PR TITLE
[ENG-6328] Show hide cols

### PR DESCRIPTION
**Pull request by futa-ikeda**
-   Ticket: [ENG-6328]
-   Feature flag: n/a

## Purpose
- Allow user to show and hide certain columns

## Summary of Changes
- Add new dropdown to show/hide columns
  - Add logic to support show/hide columns
- Tests

## Screenshot(s)
- With all columns shown
image

- With title hidden
image

### CODE REVIEW

File: app/institutions/dashboard/-components/object-list/template.hbs [hunk 1]
```diff
@@ -13,13 +13,49 @@ as |list|>
                         <span local-class='total-object-number'>{{list.totalResultCount}}</span>
                         {{t 'institutions.dashboard.object-list.total-objects' objectType=@objectType}}
                     </span>
-                    <div local-class='top-bar-button-wrapper'>
+                    <div local-class='top-bar-button-wrapper'>
```
Comment (brianjgeiger) [inline app/institutions/dashboard/-components/object-list/template.hbs:19]: Is that extra whitespace added to the end of that line?

File: app/institutions/dashboard/-components/object-list/component-test.ts [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        // Open the filter facet and load the values and select the first filter value
+        await click('[data-test-filter-facet-toggle]');
+        await click('[data-test-filter-facet-value] button');
+
+        assert.dom('[data-test-active-filter]').exists({ count: 1 }, '1 filter active');
+        assert.dom('[data-test-remove-active-filter]').exists('Remove filter button exists');
+
+        await click('[data-test-remove-active-filter]');
+        assert.dom('[data-test-active-filter]').doesNotExist('Filter removed');
+    });
```
Comment (brianjgeiger) [inline app/institutions/dashboard/-components/object-list/component-test.ts:110]: Should this test include checking that the filtered out rows are no longer visible?

**Review by brianjgeiger — CHANGES_REQUESTED** [review]
What looks to be some missing test functionality, and a little whitespace.

File: app/institutions/dashboard/-components/object-list/component-test.ts (hunk 2 above)
Comment (futa-ikeda) [reply app/institutions/dashboard/-components/object-list/component-test.ts:110]: Unfortunately, our current mirage view (mirage/views/search.ts) is fairly simple and doesn't really do anything with the filter query-params. It just serves up randomized data each time. I could try to make it a little more sophisticated so it can actually filter out certain results, but I worry that it may be a bit of an undertaking for relatively little gain.

Comment (brianjgeiger) [reply app/institutions/dashboard/-components/object-list/component-test.ts:110]: Gotcha. Probably not worth it then.

**Review by brianjgeiger — APPROVED** [review]
